backend/src/main/java/codezap/member/controller/SpringDocMemberController.java

```diff
@@ -343,4 +352,44 @@ public interface SpringDocMemberController {
             )
     )
     void logout(HttpServletResponse response);
+
+
+    @SecurityRequirement(name = "쿠키 인증 토큰")
+    @Operation(summary = "회원 정보 조회", description = "회원의 정보(이메일, 닉네임)을 조회합니다.")
+    @ApiResponse(responseCode = "200", description = "회원 정보 조회 성공")
```

Comment:
```suggestion
    @ApiResponse(responseCode = "200", description = "회원 정보 조회 성공",
            content = {@Content(schema = @Schema(implementation = FindMemberResponse.class))})
```

⚡ 성공 케이스일 때 어떤 response가 반환되는지 명시해주세요